ani-cli

```diff
@@ -213,8 +220,28 @@ update_history() {
 
 download() {
     case $1 in
-        *m3u8*) ffmpeg -loglevel error -stats -i "$1" -c copy "$download_dir/$2.mp4" ;;
-        *) aria2c --check-certificate=false --continue --summary-interval=0 -x 16 -s 16 "$1" --dir="$download_dir" -o "$2.mp4" --download-result=hide ;;
+        *m3u8*)
+            if [ -n "$output" ]; then
+                nameformat=$("echo "$output" | sed 's/%E/$ep_no/' | sed 's/%E/$seasonnum/'") # Replace the 
```

Comment:
- quotes should be outside of the brackets `"(..)"` instead of `("...")`
- use printf instead of echo
- chain sed commands using `;` at the end of one of them -> `sed 's/%E/$ep_no/;s/%E/$seasonnum/'`